parsing/TwoZones.mo

```diff
@@ -0,0 +1,198 @@
+    controllerType=Modelica.Blocks.Types.SimpleController.P,
+    k=2000,
+    yMin=0,
+    yMax=100000)
+    "Feedback controller of south zone for the heater based on room temperature"
+    annotation (Placement(transformation(extent={{-70,-40},{-50,-20}})));
+  IBPSA.Utilities.IO.SignalExchange.Overwrite oveActSou(u(
+      unit="W",
+      min=-10000,
+      max=10000), description="Heater thermal power of south zone")
+    "Overwrite the heating power of south zone"
+    annotation (Placement(transformation(extent={{-40,-40},{-20,-20}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow preHeaSou
+    "Set the heating power to the south zone"
+    annotation (Placement(transformation(extent={{0,-40},{20,-20}})));
+  Modelica.Blocks.Math.Gain effSou(k=1/0.99) "Heater efficiency in south zone"
+    annotation (Placement(transformation(extent={{0,-90},{20,-70}})));
+  IBPSA.Utilities.IO.SignalExchange.Read TRooAirSou(
+    y(unit="K"),
+    KPIs=IBPSA.Utilities.IO.SignalExchange.SignalTypes.SignalsForKPIs.AirZoneTemperature,
+    description="Zone air temperature of south zone",
+    zone=zonSouName)   "Read the room air temperature of south zone"
+    annotation (Placement(transformation(extent={{80,-60},{60,-40}})));
+
+  IBPSA.Utilities.IO.SignalExchange.Read PHeaSou(
+    y(unit="W"),
+    KPIs=IBPSA.Utilities.IO.SignalExchange.SignalTypes.SignalsForKPIs.GasPower,
+    description="Heater power of south zone")
+                                "Read the heater power consumed in south zone"
+    annotation (Placement(transformation(extent={{80,-90},{100,-70}})));
+
+  Modelica.Blocks.Math.Abs absSou
+    annotation (Placement(transformation(extent={{30,-90},{50,-70}})));
+  IBPSA.Utilities.IO.SignalExchange.Read CO2RooAirSou(
+    y(unit="ppm"),
+    KPIs=IBPSA.Utilities.IO.SignalExchange.SignalTypes.SignalsForKPIs.CO2Concentration,
+    description="Zone air CO2 concentration of south zone",
+    zone=zonSouName)   "Read the room air CO2 concentration in south zone"
+    annotation (Placement(transformation(extent={{160,-40},{180,-20}})));
+
+  Modelica.Blocks.Sources.Sine conCO2Sou(
+    amplitude=250,
+    freqHz=1/(3600*24),
+    offset=750) "Concetration of CO2 in south zone"
+    annotation (Placement(transformation(extent={{120,-40},{140,-20}})));
+  Modelica.Thermal.HeatTransfer.Components.HeatCapacitor capNor(C=1e6)
+    "Thermal capacitance of north zone"
+    annotation (Placement(transformation(extent={{30,60},{50,40}})));
+  Modelica.Thermal.HeatTransfer.Components.ThermalResistor resNor(R=0.01)
+    "Thermal resistance of north zone to outside"
+    annotation (Placement(transformation(extent={{0,50},{20,70}})));
+  Modelica.Thermal.HeatTransfer.Sensors.TemperatureSensor senTZonNor
+    "Room air temperature sensor of north zone"
+    annotation (Placement(transformation(extent={{60,50},{80,70}})));
+  Modelica.Blocks.Sources.Step setNor(
+    height=2,
+    offset=273.15 + 20,
```

Comment:
@JavierArroyoBastida Suggest change offset just to match setpoints.csv to 21 C instead of 20 C for north zone.